src/Order/Items/Feature.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Order\Items;
+
+use JetBrains\PhpStorm\ArrayShape;
+
+class Feature
+{
+    public function __construct(
+        private string $key,
+        private string|int|float $value,
```

Comment:
I believe value can also be boolean:
```suggestion
        private string|int|float|bool $value,
```